### ISSUE TITLE
Unable to rip custom tumblr URLs

### ISSUE THREAD

**Issue by poiuyrt546** [4pr0n/ripme#522]
Trying to rip `b-authentique.com`

Adding `.tumblr.com` doesn't work either.

Any clues how to get around this?

Thanks

**Comment by eatmars** [comment]
yeah same i tried changin the auth toqen but it doesnt worq anymore sadly. it worqed before few months just by changin the auth toqen :(

**Comment by eatmars** [comment]
Nevermind it worqed i just need to restart ripme

**Comment by poiuyrt546** [comment]
> Nevermind it worqed i just need to restart ripme

Were you able to rip `b-authentique.com`?

**Comment by metaprime** [comment]
@eatmars @poiuyrt546 I tried ripping `b-authentique.com` and `b-authentique.tumblr.com` and neither worked for me. If it's really a tumblr website, the same ripper should work but we have to teach it about that domain. The thing is, this is a very one-off kind of thing. Nothing wrong with adding support but it's hard to maintain this kind of thing.

I'll give it a try to see if I can easily make the ripper try to rip that website as tumblr.

**Comment by poiuyrt546** [comment]
> Ripme uses the tumblr API to get images from tumblr blogs, so if this doesn't work it either means the username is wrong, the tumblr ripper is broken, or something like permission is denied.

`b-authentique.tumblr.com/archive` shows a standard archive page for this website so the username seems to correct; unless I'm missing something here.

Other tumblr websites with non-custom URLs are still working fine for me.

**Comment by Hrxn** [comment]
Well, http://b-authentique.tumblr.com should work as any other blog on Tumblr.
The API response does not indicate anything unusual.
```JSON
{
  "meta": {
    "status": 200,
    "msg": "OK"
  },
  "response": {
    "blog": {
      "ask": false,
      "ask_anon": false,
      "ask_page_title": "Ask me anything",
      "can_subscribe": false,
      "description": "Inspiring photographers and models from all over the world.",
      "is_adult": false,
      "is_nsfw": true,
      "name": "b-authentique",
      "posts": 4541,
      "reply_conditions": "3",
      "share_likes": false,
      "subscribed": false,
      "title": "B-Authentique | Online Magazine",
      "total_posts": 4541,
      "updated": 1494423998,
      "url": "http://www.b-authentique.com/"
    }
  }
}
```
(https://api.tumblr.com/console/calls/blog/info)
